res/values-cs/strings.xml

```diff
@@ -43,11 +43,11 @@
   <string name="pref_autocapitalisation_title">Automatická kapitalizace</string>
   <string name="pref_autocapitalisation_summary">Stiskne Shift na začátku věty</string>
   <string name="pref_switch_input_immediate_title">Přepnout na posledně užívanou klávesnici</string>
-  <string name="pref_switch_input_immediate_summary">Jak bude klávesa pro přepnutí klávesnice reagovat</string>
+  <string name="pref_switch_input_immediate_summary">Namísto okna výběru klávesnice (u klávesy pro změnu klávesnice)</string>
```

Comment:
This might be too long on some devices.

Reply:
I rephrased it so it's even clearer what it does.

Should i revert this line and create new PR?

Reply:
You can modify this branch. You also need to pull the master branch. Happy to do it if the conflicts I created are too annoying to fix.

Reply:
I see there were more changes after the translation so I manually updated the file and I'm enclosing it here
[values-cs.zip](https://github.com/Julow/Unexpected-Keyboard/files/12283274/values-cs.zip)

Reply:
I've pushed directly to your branch, thanks a lot and sorry for the conflicts.

Reply:
no problem, thank you for the progress :)